development/dev.env

```
@@ -1,17 +1,19 @@
-ALLOWED_HOSTS=*
-BANNER_TOP="Local"
-CHANGELOG_RETENTION=0
-DEBUG=True
-MAX_PAGE_SIZE=0
-METRICS_ENABLED=True
-NAPALM_TIMEOUT=5
-NAUTOBOT_ROOT=/opt/nautobot
-POSTGRES_DB=nautobot
-POSTGRES_HOST=postgres
-POSTGRES_USER=nautbot
-REDIS_HOST=redis
-REDIS_PORT=6379
-# REDIS_SSL=True
+NAUTOBOT_ALLOWED_HOSTS=*
+NAUTOBOT_CHANGELOG_RETENTION=0
+NAUTOBOT_CONFIG=/opt/nautobot/nautobot_config.py
+NAUTOBOT_DB_HOST=postgres
+NAUTOBOT_DB_NAME=nautobot
+NAUTOBOT_DB_USER=nautobot
+NAUTOBOT_DB_TIMEOUT=300
+NAUTOBOT_DEBUG=True
+NAUTOBOT_NAPALM_TIMEOUT=5
+NAUTOBOT_REDIS_HOST=redis
+NAUTOBOT_REDIS_PORT=6379
 # Uncomment REDIS_SSL if using SSL
-SUPERUSER_EMAIL=admin@example.com
-SUPERUSER_NAME=admin
+# NAUTOBOT_REDIS_SSL=True
+
+# Needed for Postgres, must match the values for Nautobot above
+POSTGRES_DB=nautobot
+POSTGRES_USER=nautobot
```

Should probably move these to the following if possible (not required for this PR)
```suggestion
POSTGRES_DB=$NAUTOBOT_DB_NAME
POSTGRES_USER=$NAUTOBOT_DB_USER
```